src/rust/src/x509/csr.rs

```diff
@@ -424,15 +424,8 @@ fn create_x509_csr(
     }
 
     for py_attr in builder.getattr("_attributes")?.iter()? {
-        let (py_oid, value): (&pyo3::PyAny, &[u8]) = py_attr?.extract()?;
+        let (py_oid, value, tag): (&pyo3::PyAny, &[u8], u8) = py_attr?.extract()?;
         let oid = py_oid_to_oid(py_oid)?;
-        let tag = if std::str::from_utf8(value).is_ok() {
```

Comment:
This removes the validation even when tag _is_ Utf8String, I think we'd still want to keep that?